docs/src/layouts/MainLayout.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <q-layout view="hHh lpR fFf">
+
+    <q-header class="bg-lp-dark homepage-header">
+      <q-toolbar>
+        <q-toolbar-title>
+          <q-avatar>
+            <img src="https://cdn.quasar.dev/logo-v2/svg/logo-mono-white.svg">
+          </q-avatar>
+          QUASAR
+        </q-toolbar-title>
+        <q-space />
+
+        <q-tabs v-model="tab" active-color="white" class="homepage-header__navigation" dense indicator-color="transparent" shrink>
```

Comment:
I'm not opposed to use `q-tabs` for navigation, but in this case each of these buttons will bring to another page, so I think it's better to go with normal buttons with a `to` prop instead